queries.sql

```diff
@@ -51,14 +52,8 @@ COMMIT;
 SELECT COUNT(*) FROM animals;
 SELECT COUNT(escape_attempts) FROM animals WHERE escape_attempts = 0;
 SELECT AVG(weight_kg) FROM animals;
-SELECT neutered, MAX(escape_attempts) as escape_attempts FROM animals
- GROUP BY neutered ORDER BY MAX(escape_attempts) DESC;
-SELECT neutered, MAX(weight_kg) as max_weight , MIN(weight_kg) as min_weight FROM animals
- GROUP BY neutered;
-SELECT species, MAX(weight_kg) as max_weight , MIN(weight_kg) as min_weight FROM animals
- GROUP BY species;
-SELECT species, AVG(escape_attempts) as avg_escape_attempts FROM animals
-WHERE date_of_birth >= '1990-01-01' AND  date_of_birth < '2000-01-01' GROUP BY species;
-SELECT neutered, AVG(escape_attempts) as avg_escape_attempts FROM animals
- WHERE date_of_birth >= '1990-01-01' AND  date_of_birth < '2000-01-01' GROUP BY neutered;
- 
\ No newline at end of file
+SELECT neutered, MAX(escape_attempts) as escape_attempts FROM animals GROUP BY neutered ORDER BY MAX(escape_attempts) DESC;
+SELECT neutered, MAX(weight_kg) as max_weight , MIN(weight_kg) as min_weight FROM animals GROUP BY neutered;
+SELECT species, MAX(weight_kg) as max_weight , MIN(weight_kg) as min_weight FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) as avg_escape_attempts FROM animals WHERE date_of_birth >= '1990-01-01' AND  date_of_birth < '2000-01-01' GROUP BY species;
```

Comment:
-  Hey, this issue is not yet resolved as pointed out by the previous reviewer. Your date should include the last day of the month based on the number of days in that particular month and the last month of that year. it should be between this range:
`'1990/01/01' AND date_of_birth <= '2000/12/31'`
kindly update your query.

Reply:
@zieeco do you understand the query? i don't think you don't under the query.
What is the average number of escape attempts per animal type of those born between 1990 and 2000?